rotkehlchen/data/all_assets.json

```diff
@@ -565,11 +565,21 @@
     },
     "ANT": {
         "coingecko": "aragon",
-        "ethereum_address": "0xa117000000f279d81a1d3cc75430faa017fa5a2e",
+        "ethereum_address": "0x960b236A07cf122663c4303350609A66A7B288C0",
         "ethereum_token_decimals": 18,
         "name": "Aragon",
         "started": 1493942400,
         "symbol": "ANT",
+        "type": "ethereum token",
+        "swapped_for": "ANTv2"
+    },
+    "ANTv2": {
+        "coingecko": "aragon",
```

Comment:
Ah sorry for not mentioning it before @bingen. This will also need a cryptocompare mapping. Which is `ANT`. If there is no mapping we always try the key, but in this case "ANTv2" would not give any cryptocompare results.

So just add a `"cryptocompare": "ANT"` attribute. Since the file changes, the md5 would also need to change.